### ISSUE TITLE
CSS matrix3d Transform with a base-ten exponent number in scientific notation fails

### ISSUE THREAD

**Issue by holger-ubivent** [validator/validator#1584]
Source being tested:

```html
<div class="hallDoorTitle1" id="hall-door-1-title1" style="transform-origin: left top 0px; transform: matrix3d(0.48511, 0, 0, 0, -0.02945, 0.58367, 0, -9e-05, 0, 0, 1, 0, 317, 330, 0, 1) translateX(120.789px) translateY(0px); color: rgb(255, 255, 255);">Not yet open</div>
```

Problem seems to be the `-9e-05` in the matrix3d, which should be allowed: 

https://drafts.csswg.org/css-values-3/#number-value

```
Error: CSS: transform: Unknown dimension.
```

Also:

```
Error: CSS: transform: too few values for the property transform.
```

Would be great if you could take a look at this issue.

**Comment by sideshowbarker** [comment]
I think this is an upstream issue with the CSS-checker backend, so I’m pinging @ylafon to find out if maybe it’s a known issue, and maybe an GitHub issue has already been opened for it in the CSS checker’s issue tracker.

**Comment by ylafon** [comment]
Yes, this is related to https://github.com/w3c/css-validator/issues/366
